fimp/topic_format.md

```diff
@@ -8,21 +8,22 @@ The segments are separated by a forward slash `/` which allows for selective sub
 
 Following is the list of all topic segments and their description:
 
-| Segment | Name             | Examples         | Description                                                                                       |
-|---------|------------------|------------------|---------------------------------------------------------------------------------------------------|
-| `pt`    | Parser type      | `j1`             | Possible values are: `j1` (JSON v1), `j1c1` (JSON v1 Compression Type 1 gzip).                    |
-| `mt`    | Message type     | `evt`            | Possible values are: `cmd` (command), `evt` (event), `rsp` (response).                            |
-| `rt`    | Resource type    | `dev`            | Possible values are defined in [resource types](#resource-types) section.                         |
-| `rn`    | Resource name    | `zw`             | An arbitrary name of the application or component, preferably snake_case.                         |
-| `ad`    | Resource address | `1`              | An arbitrary string defining address of the resource, resources on the hub will normally use `1`. |
-| `sv`    | Service name     | `out_bin_switch` | Name of the device service.                                                                       |
-| `ad`    | Service address  | `1_1`            | An arbitrary string defining address of the device service.                                       |
+| Segment | Name             | Examples         | Description                                                                                         |
+|---------|------------------|------------------|-----------------------------------------------------------------------------------------------------|
+| `pt`    | Parser type      | `j1`             | Possible values are: `j1` (JSON v1), `j1c1` (JSON v1 Compression Type 1 gzip).                      |
+| `mt`    | Message type     | `evt`            | Possible values are: `cmd` (command), `evt` (event), `rsp` (response).                              |
+| `rt`    | Resource type    | `dev`            | Possible values are defined in [resource types](#resource-types) section.                           |
+| `rn`    | Resource name    | `zw`             | An arbitrary name of the application or component, preferably snake_case.                           |
+| `ad`    | Resource address | `1`              | An arbitrary string defining address of the resource, resources on the hub will normally use `1`.   |
+| `sv`    | Service name     | `out_bin_switch` | Name of the device service.                                                                         |
+| `ad`    | Service address  | `3`, `3_1`       | A string defining address of the device service. Must follow `{THING_ADDRESS}[_{GROUP_ID}]` format. |
```

Comment:
In zigbee-ad I expect always %d_%d format -> provided resourceType is 'dev'
Sending '%d' possible on 'ad' resource type.

Reply:
Yes it is.